RESTserver/src/main/java/com/softserve/if072/restservice/service/StoreService.java

```diff
@@ -1,18 +1,59 @@
 package com.softserve.if072.restservice.service;
 
 import com.softserve.if072.common.model.Store;
-import java.util.List;
-
-public interface StoreService {
-
-    List<Store> getAll();
+import com.softserve.if072.restservice.dao.mybatisdao.StoreDAO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
 
-    Store getByID(int id);
-
-    void insert(Store store);
-
-    void update(Store store);
+import java.util.List;
 
-    void delete(int id);
+@Service
+public class StoreService {
+
+    private final StoreDAO storeDAO;
+
+    @Autowired
+    public StoreService(StoreDAO storeDAO) {
+        this.storeDAO = storeDAO;
+    }
+
+    @Transactional
+    public List<Store> getAllStores() {
+        List<Store> stores = storeDAO.getAll();
+        if (!stores.isEmpty()){
+            return stores;
+        } else {
+            throw new RuntimeException("Stores not found");
```

Comment:
'Stores not found' move to constants, because it is used more than in one place.